modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -546,6 +556,18 @@ GAPI_FLUID_KERNEL(GFluidBoxFilter, cv::gapi::imgproc::GBoxFilter, true)
     {
         return { borderType, borderValue};
     }
+
+    static int getWindow(const cv::GMatDesc& /* src */,
```

Comment:
Are there some tests for added functionality? As I can see GFluidBlur still has static window defined so this callback will never be called

Reply:
Applied.